mono/metadata/w32mutex-unix.c

```diff
@@ -37,27 +37,31 @@ static gpointer
 mono_w32mutex_open (const char* utf8_name, gint32 rights G_GNUC_UNUSED, gint32 *win32error);
 
 static void
-thread_own_mutex (MonoInternalThreadHandle internal, gpointer handle, MonoW32Handle *handle_data)
+thread_own_mutex (MonoInternalThread *internal, gpointer handle, MonoW32Handle *handle_data)
+// Thread and InternalThread are pinned/mature.
```

Comment:
don't put comments between the parameters and the `{`

We usually put a `/** apidoc */` comment before the declaration and mention `\p internal` or `\c MonoInternalThread`.